Agreements/Airport/AA_Antrim.txt

```diff
@@ -1,2 +1,20 @@
-COPX:EGAA:*:*:*:*:AAAPP:Antrim:*:9000:^BEL
-COPX:EGAC:*:*:*:*:AAAPP:Antrim:*:9000:^
+COPX:EGAA:*:PEPOD:*:*:AAAPP:Antrim:*:9000:^BEL
+COPX:EGAC:*:PEPOD:*:*:AAAPP:Antrim:*:9000:^BEL
+
+COPX:EGAA:*:RINGA:*:*:AAAPP:Antrim:*:9000:^BEL
+COPX:EGAC:*:RINGA:*:*:AAAPP:Antrim:*:9000:^BEL
+
+COPX:EGAA:*:BLACA:*:*:AAAPP:Antrim:*:9000:^BEL
+COPX:EGAC:*:BLACA:*:*:AAAPP:Antrim:*:9000:^BEL
+
+COPX:EGAA:*:NUMPI:*:*:AAAPP:Antrim:*:9000:^BEL
+COPX:EGAC:*:NUMPI:*:*:AAAPP:Antrim:*:9000:^BEL
+
+COPX:EGAA:*:DEGOS:*:*:AAAPP:Antrim:*:9000:^BEL
+COPX:EGAC:*:DEGOS:*:*:AAAPP:Antrim:*:9000:^BEL
+
+COPX:EGAA:*:STN:*:*:AAAPP:Antrim:*:9000:^BEL
+COPX:EGAC:*:STN:*:*:AAAPP:Antrim:*:9000:^BEL
+
+COPX:EGAA:*:MAC:*:*:AAAPP:Antrim:*:9000:^BEL
+COPX:EGAC:*:MAC:*:*:AAAPP:Antrim:*:9000:^BEL
```

Comment:
These agreements should all be between AAAPP and the Rathlin sector I think... will check with Caspian!

Please create a new file called AA_Rat.txt for these agreements

I think the STN agreement should also use STN as 'FIXAFTER', just because it is so far away from EGAA:

```
COPX:EGAA:*:*:STN:*:AAAPP:Rathlin:*:9000:^BEL
COPX:EGAC:*:*:STN:*:AAAPP:Rathlin:*:9000:^BEL
```

Reply:
Good idea, Hold up a second